docs/tutorials/markdown-guide.md

```diff
@@ -0,0 +1,199 @@
+| [Ordered List](#ordered-list) | 1. Item <br /> 2. Item <br /> 3. Item  |
+| [Unordered List](#unordered-list) | - Item <br /> - Item <br /> - Item |
+| [Horizontal Rule](#horizontal-rule) | --- |
+| [Link](https://www.markdownguide.org/basic-syntax/#links) | `[title](https://www.example.com)` |  
+| [Image](https://www.markdownguide.org/basic-syntax/#images-1) | `![text](image.jpg)` |
+|[Line Break](#line-break) | text + double space |
+| [Comments](#comments) | `<!---Comment--->` |
+
+
+> Can get up to 6 different headings depending on the number of `#`
+
+## Structure Syntax
+
+
+### **Unordered List**  
+
+Unordered list syntax
+
+```markdown title="Markdown"
+    - First Item
+        - Sub Item
+    - Second Item
+    - Third Item  
+```
+**Example:**
+- First Item
+    - Sub Item
+- Second Item
+- Third Item
+
+>[more information](https://www.markdownguide.org/basic-syntax/#unordered-lists)
+
+### **Ordered List**
+Ordered list syntax
+
+```markdown title="Markdown"
+    1. First Item
+        1. Sub Item
+    2. Second Item
+    3. Third Item
+```
+**Example:**
+
+1. First Item
+    1. Sub Item
+2. Second Item
+3. Third Item
+
+>[more information](https://www.markdownguide.org/basic-syntax/#ordered-lists)
+
+### **Table**
+
+Table syntax
+
+```markdown title="Markdown"
+| Syntax | Description |
+| ----------- | ----------- |
+| Header | Title |
+| Paragraph | Text |
+```
+**Example:**  
+
+| Syntax | Description |
+| ----------- | ----------- |
+| Header | Title |
+| Paragraph | Text |
+
+>[more information](https://www.markdownguide.org/extended-syntax/#tables)
+
+### **Check List** 
+
+Check list syntax
+
+```markdown title="Markdown"
+    - [x] First option
+    - [ ] Second option
+    - [ ] Third option
+```
+
+**Example:**
+
+- [x] First option
+- [ ] Second option
+- [ ] Third option
+
+>[more information](https://www.markdownguide.org/extended-syntax/#task-lists)
+
+### **Line Break**
+
+Line break syntax
+
+```markdown title="Markdown"
+    Text  <!---Space Space--->
+    another Text
+```
+
+**Example:**
+
+Text  
+Another Text
+
+### **Comments **
+
+
+**Example:**
+
+```markdown title="Markdown"
+    [comment]: <> (This is a comment, it will not be included)
+
+    [//]: <> (This is also a comment.)
+
+    [//]: # (This may be the most platform independent comment)`
+```
+
+## Another Syntaxes
+
+### **Links**
+
+Links syntax
+
+```markdown title="Markdown"
+[Eos Costa Rica](https://es.eoscostarica.io)
+```
+**Example:**
+[Eos Costa Rica](https://es.eoscostarica.io)
+
+### **Images**
+
+Images syntax
+
+```markdown title="Markdown" 
+![](https://media-exp1.licdn.com/dms/image/C4E0BAQFBDxX8sb8eRQ/company-logo_200_200/0/1562012485976?e=2159024400&v=beta&t=gifOwJQ3a5xmVcAzmORT7thKl9uzaKH9bv74yp2vHns)
+```
+**Example:**  
+![](https://media-exp1.licdn.com/dms/image/C4E0BAQFBDxX8sb8eRQ/company-logo_200_200/0/1562012485976?e=2159024400&v=beta&t=gifOwJQ3a5xmVcAzmORT7thKl9uzaKH9bv74yp2vHns)
```

Comment:
Everything looks good to me, but I think that as an example, could be used an image without the black background because all the content is a combination of whites, grays and black, but this last one is just for text. It just a suggestion.